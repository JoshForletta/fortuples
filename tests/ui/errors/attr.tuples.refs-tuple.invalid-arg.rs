use fortuples::fortuples;

struct Container<T>(T);

fortuples! {
    #[tuples::refs_tuple(A)]
    impl Container<#Tuple> {}
}

fn main() {}
