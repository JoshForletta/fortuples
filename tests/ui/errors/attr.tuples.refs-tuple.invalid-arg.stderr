error: unexpected arguments
 --> tests/ui/errors/attr.tuples.refs-tuple.invalid-arg.rs:6:25
  |
6 |     #[tuples::refs_tuple(A)]
  |                         ^^^
